content/_data/changelogs/weekly.yml

```
@@ -8752,6 +8752,54 @@
   # pull: 4900 (PR title: Document read only inputs)
   # pull: 4906 (PR title: Fix invalid translations in ApiTokenProperty)
 
+- version: '2.254'
+  date: 2020-08-25
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4912
+    issue: 63458
+    authors:
+    - jglick
+    message: |-
+      Revert an earlier bug fix that caused several varieties of deadlocks (regression in 2.231).
```

This is not very informative


```suggestion
      Stop pre-formatting agent logs to prevent deadlocks (regression in 2.231).
```